sys/src/9/amd64/vq9p.c

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
yeah. We've just finished a port of virtio to akaros and there's almost nothing architecture dependent in it. Is there anything in your code that is x86 specific? I don't see anything.